doc/src/site/sphinx/ml/sw_stacked_ensemble.rst

```diff
@@ -0,0 +1,202 @@
+Train Stacked Ensemble Model in Sparkling Water
+-----------------------------------------------
+
+Stacked Ensemble is a supervised machine learning algorithm that finds an optimal combination of a collection
+of prediction algorithms (base models). For further details about the algorithm and its parameters see `H2O-3 documentation
+<https://docs.h2o.ai/h2o/latest-stable/h2o-docs/data-science/stacked-ensembles.html>`__.
+
+
+Sparkling Water provides API in Scala and Python for Stacked Ensemble. The following sections describe how to
+utilize Stacked Ensemble in both languages. See also :ref:`parameters_H2OStackedEnsemble`.
+
+.. |start cluster| replace:: Start H2O cluster inside the Spark environment
+
+.. |get data| replace:: Parse the data using H2O and convert them to Spark Frame
+
+.. |setup base algorithms| replace:: Setup the algorithms the StackedEnsemble will operate with. StackedEnsemble will
+    automatically train the corresponding (base) models and pass them to H2O backend when needed. There are currently
+    two options how a meta-learner in StackedEnsemble combines the base models. It either utilizes cross validated
+    predictions or uses a blending frame. In the former case, it's important to keep the same folding across
+    the base models and set *setKeepCrossValidationPredictions* to *true* as the cross-validated predicted values
+    will be used by meta-learner. Furthermore, as the Stacked Ensemble combines the base models inside an H2O backend
+    the base models have to be available there as well and therefore *setKeepBinaryModels* has to be set to *true* too.
+
+.. |setup algorithm and train| replace:: Then, specify the algorithms when setting up the StackedEnsemble and train it.
+
+.. |advanced usage| replace:: Inside the *fit* method, StackedEnsemble will internally create, utilize and delete
+    models corresponding to the input algorithms. If there is any reason to make use of those models,
+    call **setKeepBaseModels** with a *true* argument to retain them. Then, models can be obtained by calling
```

Comment:
It seems to me that we are talking only about one kind of model. Only binary models needs to be looked after. In SW, there is no real use case for them. StackedEnsablemle MOJO already contains MOJOs of base models. We can just expose them. 

Thus IMHO `setKeepBaseModels` and `getBaseModels` are not needed.

Reply:
Implementation was simplified and a plan is to expose base models from StackedEnsemble MOJO (as suggested) in a separate PR.